src/quantem/imaging/lattice.py

```diff
@@ -0,0 +1,2961 @@
+        ax.set_title("polarization" + (" (median subtracted)" if subtract_median else ""))
+        plt.tight_layout()
+
+        # Circular legend (same mapping and label)
+        if show_colorbar:
+            divider = make_axes_locatable(ax)
+            ax_c = divider.append_axes("right", size="28%", pad="6%")
+
+            N = 256
+            yy = np.linspace(-1, 1, N)
+            xx = np.linspace(-1, 1, N)
+            YY, XX = np.meshgrid(yy, xx, indexing="ij")
+            rr = np.sqrt(XX**2 + YY**2)
+            disk = rr <= 1.0
+
+            ang_grid = np.arctan2(XX, -YY)
+            if phase_dir_flip:
+                ang_grid = -ang_grid
+            ang_grid += np.deg2rad(phase_offset_deg)
+
+            amp_grid = np.clip(rr, 0, 1)
+            rgba_grid = array_to_rgba(amp_grid, ang_grid, chroma_boost=chroma_boost)
+            rgba_grid[~disk] = 0.0
+
+            ax_c.imshow(
+                rgba_grid, origin="lower", extent=(-1, 1, -1, 1), interpolation="nearest", zorder=0
+            )
+            ax_c.set_aspect("equal")
+            ax_c.axis("off")
+
+            ring = Circle((0, 0), 0.98, facecolor="none", edgecolor="k", linewidth=1.2, zorder=3)
+            ring.set_clip_on(False)
+            ax_c.add_patch(ring)
+
+            # Cardinal labels (down/right/up/left)
+            ax_c.text(0.00, -1.12, "0°", ha="center", va="top", fontsize=9, color="k")
+            ax_c.text(1.12, 0.00, "90°", ha="left", va="center", fontsize=9, color="k")
+            ax_c.text(0.00, 1.12, "180°", ha="center", va="bottom", fontsize=9, color="k")
+            ax_c.text(-1.12, 0.00, "270°", ha="right", va="center", fontsize=9, color="k")
+
+            # Scale arrow along +x, label centered above midpoint (white)
+            scale_len = 0.85
+            arrow_scale = FancyArrowPatch(
+                (0.0, 0.0),
+                (scale_len, 0.0),
+                arrowstyle=ArrowStyle.Simple(head_length=10.0, head_width=6.0, tail_width=2.0),
+                mutation_scale=1.0,
+                linewidth=1.2,
+                facecolor="k",
+                edgecolor="k",
+                zorder=4,
+                shrinkA=0.0,
+                shrinkB=0.0,
+            )
+            arrow_scale.set_clip_on(False)
+            ax_c.add_patch(arrow_scale)
+
+            mid_x, mid_y = scale_len / 2.0, 0.0
+            ax_c.text(
+                mid_x,
+                mid_y + 0.14,
+                f"{disp_cap_px:.2g} px",
+                ha="center",
+                va="bottom",
+                fontsize=9,
+                color="w",
+            )
+
+            # Crosshairs & generous limits to avoid clipping
+            ax_c.plot([0, 0], [-0.9, 0.9], color=(0, 0, 0, 0.15), lw=0.8, zorder=2)
+            ax_c.plot([-0.9, 0.9], [0, 0], color=(0, 0, 0, 0.15), lw=0.8, zorder=2)
+            ax_c.set_xlim(-1.35, 1.35)
+            ax_c.set_ylim(-1.25, 1.35)
+
+        return fig, ax
+
+    def plot_polarization_image(
```

Comment:
We should roughly scale the "tile" polarization plots so they match the image - don't use square tiles by default, since this leads to a stretched image:
<img width="754" height="821" alt="image" src="https://github.com/user-attachments/assets/466ed55b-696a-4c82-b68d-db71e1898494" />